lib/src/tasks/test/cli.dart

```diff
@@ -44,6 +44,11 @@ class TestCli extends TaskCli {
         negatable: true,
         defaultsTo: defaultPubServe,
         help: 'Serves browser tests using a Pub server.')
+    ..addFlag('debug',
+        abbr: 'd',
+        negatable: true,
+        defaultsTo: defaultDebug,
+        help: 'Debug browser tests using the "--pause-after-load" flag.')
```

Comment:
If this set more than one flag, I'd be fine with naming it `debug`, but since it's a 1:1 mapping I'd rather keep it consistent. Also, with v2, all unused flags/options will be passed through to the underlying executable, so it would need to be `--pause-after-load` at that point anyway.